### ISSUE TITLE
New build error with get.py with cygwin

### ISSUE THREAD

**Issue by hallard** [esp8266/Arduino#943]
If I launch get.py from windows command line it works, but from cygwin shell I've got an error
Not really an issue, just to let you know
```
 python get.py
Traceback (most recent call last):
  File "get.py", line 99, in <module>
    print('Platform: {0}'.format(identify_platform()))
  File "get.py", line 96, in identify_platform
    return arduino_platform_names[platform.system()][bits]
KeyError: 'CYGWIN_NT-10.0'
tools #
```

**Comment by igrr** [comment]
@hallard should be fixed with daf360d

**Comment by hallard** [comment]
@igrr 
Just to let you know, tested with Arduino 1.6.6 from scratch (windows10 x64 + cygwin), worked fine ;-)
